Sources/MockoloFramework/Models/VariableModel.swift

```diff
@@ -91,10 +94,19 @@ final class VariableModel: Model {
             return rxVar
         }
 
+        let modifier: Modifier
+        if let modifiers = self.modifiers,
```

Comment:
This should be moved to SwiftSyntaxExtension - 'Trivia.metadata(with annotation: String, in val: String)'